ThePhasesOfCraftship/2_best_practice_first/testingBasics/exercises/1_Red_Green_Refactor/1_1_Fizzbuzz/src/index.spec.ts

```diff
@@ -1,5 +1,59 @@
+import { fizzbuzz } from "./fizzbuzz";
 
 describe("fizzbuzz", () => {
+  it("should return string", () => {
+    expect(typeof fizzbuzz(5)).toBe("string");
+  });
 
+  it.each([3, 6, 9, 12])(
+    "should return fizz for (%i) which is multiplication of 3",
```

Comment:
The test name can be improved a little bit. Something like "Returns Fizz for multiples of 3" clearly describes the test case

Reply:
Changed to `should return buzz for multiples of 5 (%i)`